index.js

```diff
@@ -29,8 +29,11 @@ export async function upload (firstPath, opts) {
     }
   }
   spinner.start('Storing')
-  // @ts-ignore
-  const root = await client.uploadDirectory(files, {
+  /** @type {(o?: import('@web3-storage/w3up-client/src/types').UploadOptions) => Promise<import('@web3-storage/w3up-client/src/types').AnyLink>} */
+  const uploadFn = files.length === 1 && opts['no-wrap']
```

Comment:
i think files.length would be > 1 if you added
```
foo
├── alan.txt
└── bar.txt
```
is it  that directories are always wrapped so the `foo` is preserved?

we should warn the user that --no-wrap is ignored in that case, or error

Reply:
Ah, `files-from-path` is relative to working directory, not common path...